data/editor.config

```diff
@@ -991,6 +991,9 @@
       <type id="attraction-animal">
         <include field="name" />
       </type>
+      <type id="tourism-camp_pitch" group="accomodation">
+        <include group="poi_internet" />
```

Comment:
Are there camp pitches with the internet? Should this tag belong to the camp site?

Reply:
I would also think it belongs to the camp site as a whole, unless maybe only part of the facility has reach to the wifi?

In any case, FWIW, `internet_access` is in the wiki under "Tags used in combination":

https://wiki.openstreetmap.org/wiki/Tag:tourism%3Dcamp_pitch

Reply:
Yes, this is an error due to my not totally understanding what everything is and just trying to follow contextual convention -- I will remove poi_internet with my next force push (pending resolution of other questions).